src/time_tracker/invoices/handlers.clj

```diff
@@ -105,8 +119,11 @@
                       connection
                       invoice-id)]
       (do
```

Comment:
`minor` 
Same as before, you can move the `log/debug` inside of the `let` block and remove the `do` block